compute_tools/src/compute.rs

```diff
@@ -760,6 +766,10 @@ impl ComputeNode {
             handle_grants(&spec, &mut client, self.connstr.as_str())?;
             handle_extensions(&spec, &mut client)?;
             handle_extension_neon(&mut client)?;
+            // We can skip handle_migrations here because a new migration can only appear
+            // if we have a new version of the compute_ctl binary, which can only happen
+            // if compute got restarted, in which case we'll end up inside of apply_config
+            // instead of reconfigure.
```

Comment:
`handle_migrations` also creates few tables. Makes sense to do that during provisioning to have less impact on starts later on.

Reply:
Where should I put it? This is happening asynchronously anyway, so moving it probably won't impact starts much anyway